client/styles/components/_console.scss

```diff
@@ -88,11 +88,11 @@
 }
 
 .preview-console__logged-times {
-	font-size: #{10 / $base-font-size}rem;
+
 	font-weight: bold;
 	margin: #{2 / $base-font-size}rem 0 0 #{8 / $base-font-size}rem;
-	border-radius: 10px;
-	padding: #{1 / $base-font-size}rem #{4 / $base-font-size}rem;
+	border-radius: #{$base-font-size*10}px;
```

Comment:
This will not change when the font changed. `$base-font-size` is defined here: https://github.com/processing/p5.js-web-editor/blob/master/client/styles/abstracts/_variables.scss#L1, it's better to move to inline style

Reply:
oh,i got confused with name